tools/ios/MASTG-TOOL-0103.md

```diff
@@ -0,0 +1,374 @@
+---
+title: IPSW
+platform: ios
+source: https://github.com/blacktop/ipsw
+host: [windows, linux, macOS]
+---
+
+IPSW calls itself an "iOS/macOS Research Swiss Army Knife". In general, IPSW allows you to obtain iOS specific files (IPSW, OTA, ...) and also statically analyze them. For application analysis, the most interesting features are the Objective-C and Swift class-dumps. Other features are available (kernelcache parser, device-tree parser, disassembler, etc) but are only useful if you're analyzing the security of the OS itself.
+
+## Extracting Objective-C Class Information
```

Comment:
These kind of things belong in the corresponding MASTG-TECH. This way once we enable xrefs for these components we'll be able to navigate to these examples of use right from the tool.